pycloudlib/gce/cloud.py

```diff
@@ -199,10 +203,16 @@ def _find_latest_image(self, release: str, image_type: ImageType):
             )
 
         image = sorted(image_list, key=lambda x: x["creationTimestamp"])[-1]
-        self._log.debug("Found image name: %s", image["name"])
+        self._log.debug(
```

Comment:
I think we should also log the searched arch when we can't find an image in the ["Could not find generic image on arch: {arch} for {release} release"](https://github.com/canonical/pycloudlib/pull/215/files#diff-6dc804f6ffbd945a53ff2ee7e65d3882f64528486f2542b3727b24050ba0a825R199)